pkg/gateway/model_build_targetgroup.go

```diff
@@ -240,36 +250,33 @@ func (t *targetGroupModelBuildTask) BuildTargetGroup(ctx context.Context) error
 			HealthCheckConfig:   healthCheckConfig,
 			IpAddressType:       ipAddressType,
 		},
-	}
-
-	tg := latticemodel.NewTargetGroup(t.stack, tgName, tgSpec)
-	t.log.Infof("buildTargetGroup, tg[%s], tgSpec %v", tgName, tg)
-
-	// add targetgroup to localcache for service reconcile to reference
-	// for serviceexport, the httproutename is set to ""
-	t.datastore.AddTargetGroup(tgName, "", "", "", tgSpec.Config.IsServiceImport, "")
-
-	if !t.serviceExport.DeletionTimestamp.IsZero() {
-		// triggered by serviceexport delete
-		t.datastore.SetTargetGroupByServiceExport(tgName, false, false)
-	} else {
-		// triggered by serviceexport add
-		t.datastore.SetTargetGroupByServiceExport(tgName, false, true)
-	}
-
-	// for serviceexport, the routeName is null
-	dsTG, err := t.datastore.GetTargetGroup(tgName, "", false)
+	})
+	t.log.Infof("buildTargetGroup, stackTG[%s], tgSpec[%v]", targetGroupName, stackTG)
```

Comment:
1. No %v for info log.
2. Logging format consistency: use either Infow("msg", key1, val1) or Infof("msg, key1: %s")

Reply:
`tgSpec[%v]", stackTG`, stackTG has many fields, do I need to list them one by one? Code is redundant?


```
stackTG := latticemodel.NewTargetGroup(t.stack, targetGroupName, latticemodel.TargetGroupSpec{
		Name: targetGroupName,
		Type: latticemodel.TargetGroupTypeIP,
		Config: latticemodel.TargetGroupConfig{
			VpcID: config.VpcID,
			// Fill in default HTTP port as we are using target port anyway.
			Port:                80,
			IsServiceImport:     false,
			IsServiceExport:     true,
			K8SServiceName:      t.serviceExport.Name,
			K8SServiceNamespace: t.serviceExport.Namespace,
			Protocol:            protocol,
			ProtocolVersion:     protocolVersion,
			HealthCheckConfig:   healthCheckConfig,
			IpAddressType:       ipAddressType,
		},
	})
```

Reply:
why do you need log all of them into info log?

Reply:
Changed to 
```
	t.log.Infow("stackTG:",
		"targetGroupName", stackTG.Spec.Name,
		"K8SServiceName", stackTG.Spec.Config.K8SServiceName,
		"K8SServiceNamespace", stackTG.Spec.Config.K8SServiceNamespace,
		"Protocol", stackTG.Spec.Config.Protocol,
		"ProtocolVersion", stackTG.Spec.Config.ProtocolVersion,
		"IpAddressType", stackTG.Spec.Config.IpAddressType,
		"HealthCheckConfig", stackTG.Spec.Config.HealthCheckConfig,
	)
```
 
New log output:
```
{"level":"info","ts":1694460321.9547572,"logger":"controller.service-export","caller":"gateway/model_build_targetgroup.go:255","msg":"stackTG:","targetGroupName":"k8s-pegas-2-tvxttxm5bg-non-default","K8SServiceName":"pegas-2-tvxttxm5bg","K8SServiceNamespace":"non-default","Protocol":"HTTP","ProtocolVersion":"HTTP1","IpAddressType":"IPV4","HealthCheckConfig":"<nil>"}
```

Reply:
ok, still dont understand why it should be with info log.

Reply:
Changed to `t.log.Debugw`